### PR TITLE
OCD-3580:  G1/G2 Cures Criteria Data Fix

### PR DESCRIPTION
**Pull request by tmy1313**

### CODE REVIEW

File: chpl/chpl-service/src/main/java/gov/healthit/chpl/scheduler/job/measurefix/MeasureFixJob.java
Comment (kekey1) [inline chpl/chpl-service/src/main/java/gov/healthit/chpl/scheduler/job/measurefix/MeasureFixJob.java:72]: I did not know about this method! Just looked it up - I like it!

File: chpl/chpl-service/src/main/java/gov/healthit/chpl/scheduler/job/measurefix/MeasureFixJob.java
Comment (kekey1) [inline chpl/chpl-service/src/main/java/gov/healthit/chpl/scheduler/job/measurefix/MeasureFixJob.java]: It's not clear to me if/why this code needs to be wrapped in a transaction. It looks like it's calling a couple of manager methods which I expect are already marked as Transactional themselves. I don't think it would hurt anything to have this transaction here either, I'm just wondering if it's needed for another reason.

**Review by kekey1 — APPROVED** [review]

File: chpl/chpl-service/src/main/java/gov/healthit/chpl/scheduler/job/measurefix/MeasureFixJob.java
Comment (kekey1) [inline chpl/chpl-service/src/main/java/gov/healthit/chpl/scheduler/job/measurefix/MeasureFixJob.java]: I know we agreed not to log activity so we didn't have all those activities as a side-effect of this job. I'm also now thinking that this will change the lastUpdated date for each of these listings (even ones that are valid and won't have their g1/g2 measures changed at all) and I just want to make sure that we don't mind having that happen. Something to ask at the feature review maybe but I thought of it now and wanted to write it down.

**Review by kekey1 — DISMISSED** [review]

File: chpl/chpl-resources/src/main/resources/jobs.xml
Comment (andlar) [inline chpl/chpl-resources/src/main/resources/jobs.xml]: Our plan is to address these alerts, right? I don't see a reason to hold up the PR now, but maybe before it's merged?

**Review by andlar — DISMISSED** [review]

**Review by kekey1 — DISMISSED** [review]

**Review by kekey1 — APPROVED** [review]

**Review by andlar — APPROVED** [review]

**Pull request merged**
